06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -207,17 +212,18 @@ export default class InvertedIndex {
 
   /**
    * @cleanValues method
-   * @param {string} word - word to clean
-   * @returns {Array} contains cleaned words
+   * @param {string} words - word to clean
+   * @returns {Array} words contains cleaned words
    * cleans the keyword for search
    */
-static cleanValues(words) {
+  static cleanValues(words) {
     let value = words.replace(/[&\\#,+()$~%.'":*?<>\-^!@{}]/g, '')
       .replace(/[[|\]]/g, '')
       .toLowerCase()
       .trim()
       .split(/\b\s+(?!$)/);
-    value = value.filter(word => word !== '');
+      
```

Comment:
Please check the trailing spaces and indentations in this your cleanValues() method.